### PR TITLE
refactor(module:progress, rate): refactor progress, rate OnPush

### PR DESCRIPTION
**Pull request by billpeace**
#2381 

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: components/rate/nz-rate.spec.ts [hunk 1]
```diff
@@ -11,146 +11,148 @@ import { NzRateModule } from './nz-rate.module';
 describe('rate', () => {
   beforeEach(fakeAsync(() => {
     TestBed.configureTestingModule({
-      imports     : [ NzRateModule, FormsModule, ReactiveFormsModule ],
```
Comment (vthinkxie) [inline components/rate/nz-rate.spec.ts]: plz do not reformat code style

Comment (billpeace) [reply components/rate/nz-rate.spec.ts]: OK

**Comment by wendzhue** [comment]
@billpeace Ping~ Any update on this?

**Comment by wendzhue** [comment]
This is going to be closed for inactivity. Please consider open another PR if you are going to make other changes.

**Pull request closed without merging**